tests/test_vagrant.py

```diff
@@ -269,7 +269,10 @@ def test_vm_status(vm_dir):
 
 
 def test_vm_lifecycle(vm_dir):
-    """Test methods controlling the VM - init(), up(), halt(), destroy()."""
+    """Test methods controlling the VM - init(), up(), suspend(), halt(), destroy()."""
```

Comment:
What was the reason for removing the suspend() here ?

Reply:
I didn't? I just updated the comment to include `suspend()`.
```python
    v.suspend()
    assert v.SAVED == v.status()[0].state
```